src/main/java/core/basesyntax/Engine.java

```diff
@@ -1,5 +1,31 @@
 package core.basesyntax;
 
-public class Engine {
+public class Engine implements Cloneable {
+    private double volume;
+    private double maxPower;
+    private double energyCapacity;
+    private EngineType engineType;
+    private double distanceToService;
+    private boolean check;
+
+    public Engine() {
```

Comment:
just pass these parameters to constructor

Reply:
done